pinot-core/src/test/java/org/apache/pinot/index/readerwriter/FixedByteChunkSVForwardIndexReaderWriterTest.java

```diff
@@ -276,56 +257,6 @@ public void testDouble(ChunkCompressorFactory.CompressionType compressionType)
     FileUtils.deleteQuietly(outFileEightByte);
   }
 
-  public void testBytes(ChunkCompressorFactory.CompressionType compressionType)
```

Comment:
This test was for fixed length bytes which for some reason FixedByteChunk writer also supported.

Reply:
It can be supported, but is never used. STRING and BYTES are always handled as var-bytes (we don't have the metadata to denote whether a column is fixed-bytes or var-bytes).